src/test/java/de/tum/in/www1/artemis/assessment/ParticipantScoreIntegrationTest.java

```diff
@@ -284,16 +248,14 @@ void getCourseScores_asInstructorOfCourseWithGradedPresentations_shouldReturnCou
     void getExamScores_asInstructorOfCourse_shouldReturnExamScores() throws Exception {
         List<ScoreDTO> courseScores = request.getList("/api/exams/" + idOfExam + "/exam-scores", HttpStatus.OK, ScoreDTO.class);
         assertThat(courseScores).hasSize(1);
-        ScoreDTO scoreOfStudent1 = courseScores.stream().filter(scoreDTO -> scoreDTO.studentId.equals(idOfStudent1)).findFirst().orElseThrow();
+        ScoreDTO scoreOfStudent1 = courseScores.stream().filter(scoreDTO -> scoreDTO.studentId.equals(student1.getId())).findFirst().orElseThrow();
         assertThat(scoreOfStudent1.studentLogin).isEqualTo(TEST_PREFIX + "student1");
         assertThat(scoreOfStudent1.pointsAchieved).isEqualTo(5.0);
         assertThat(scoreOfStudent1.scoreAchieved).isEqualTo(5.6);
         assertThat(scoreOfStudent1.regularPointsAchievable).isEqualTo(90.0);
     }
 
-    private TextExercise createIndividualTextExerciseForExam() {
-        Exam exam;
-        exam = examRepository.findWithExerciseGroupsAndExercisesById(idOfExam).orElseThrow();
+    private TextExercise createIndividualTextExerciseForExam(Exam exam) {
```

Comment:
Would it make sense to extract this method as well?

Reply:
I completely removed this method and used an already existing one